Umbraco-Cloud/Set-Up/Config-Transforms/index.md

```diff
@@ -4,7 +4,7 @@ versionFrom: 9.0.0
 
 # Config Transforms
 
-In this article you can learn how to use config transform files to apply environment specific configuration and settings to your Umbraco Cloud project.
+In this article you can learn how to use config transform files to apply environment specific configuration and settings to your Umbraco Cloud project using both Web.Config files as well as the appSettings.json file.
```

Comment:
Sounds good, i will look into it :) 

The reason for removing the transforms for packages is because it is not working as reported here: https://github.com/umbraco/Umbraco.Cloud.Issues/issues/568 :-)